miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniAppSdkConfig.kt

```diff
@@ -13,18 +13,18 @@ data class MiniAppSdkConfig(
     val baseUrl: String,
     val rasAppId: String,
     val subscriptionKey: String,
-    val hostAppVersionId: String,
+    val hostAppVersionId: String = "",
     val hostAppUserAgentInfo: String,
     val isPreviewMode: Boolean
 ) {
-    internal val key = "$baseUrl-$isPreviewMode-$rasAppId-$subscriptionKey-$hostAppVersionId"
+    internal val key = "$baseUrl-$isPreviewMode-$rasAppId-$subscriptionKey"
 
     @Deprecated("Use isPreviewMode instead of isTestMode")
     constructor(
         baseUrl: String,
         rasAppId: String,
         subscriptionKey: String,
-        hostAppVersionId: String,
+        hostAppVersionId: String = "",
```

Comment:
Do we need to add it in secondary constructor? We haven't released this constructor yet for any existing client. I think, we could remove it. But, with the primary constructor, we can add a `@Deprecated` annotation e.g.

```
data class MiniAppSdkConfig(
    ...
    @Deprecated("It has been removed from api endpoints")
    val hostAppVersionId: String = "",
    ...
) ...
```

Reply:
No, the only change is I put the default value so nothing is deprecated here.